From a50abcaade040e7545730c3fa658550225d4ec3d Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Wed, 3 Apr 2013 12:59:23 +0800
Subject: [PATCH 185/281] ENGR00232210 BT:fix one bt tweak

Restore not enable BT default and delete obsolete propertity.
For it is confilict for PCIE_DIS_B. 3G also use it which will
cause 3G not work.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 sabresd_6dq/BoardConfig.mk | 2 +-
 sabresd_6dq/init.rc        | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/sabresd_6dq/BoardConfig.mk b/sabresd_6dq/BoardConfig.mk
index 22bdec8..cd5d35f 100755
--- a/sabresd_6dq/BoardConfig.mk
+++ b/sabresd_6dq/BoardConfig.mk
@@ -79,7 +79,7 @@ $(error "TARGET_USERIMAGES_USE_UBIFS and TARGET_USERIMAGES_USE_EXT4 config open
 endif
 endif
 
-BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0:dev=ldb,bpp=32 video=mxcfb1:off video=mxcfb2:off fbmem=10M fb0base=0x27b00000 vmalloc=400M androidboot.console=ttymxc0 androidboot.hardware=freescale bluetooth
+BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init video=mxcfb0:dev=ldb,bpp=32 video=mxcfb1:off video=mxcfb2:off fbmem=10M fb0base=0x27b00000 vmalloc=400M androidboot.console=ttymxc0 androidboot.hardware=freescale
 
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
 #UBI boot command line.
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index adb9f95..5f105d1 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -99,7 +99,6 @@ on boot
     setprop ro.lightsensor.threshold  20
 
     # No bluetooth hardware present
-    setprop hw.bluetooth 0
     setprop wlan.interface wlan0
 
 # mount the debugfs
-- 
1.8.0

